deploy/kubernetes/overlays/alpha/kustomization.yaml

```diff
@@ -2,17 +2,15 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 bases:
 - ../stable
-patches:
-- controller_add_snapshotter.yaml
+images:
+- name: gke.gcr.io/csi-snapshotter
```

Comment:
I think we can remove support for the alpha snapshot feature.

Reply:
Can you please clarify @msau42 ? Do you mean use beta snapshot side car here for the alpha overlay?

Reply:
Ack